tests/test-private-upload.ts

```diff
@@ -0,0 +1,128 @@
+import { GetObjectCommand, S3Client, S3ClientConfig } from '@aws-sdk/client-s3';
+import { getSignedUrl } from '@aws-sdk/s3-request-presigner';
+import { test, expect } from '@jest/globals';
+import { MiddlewareType } from '@smithy/types';
+
+import { handler } from '../src/lambda-handler';
+import { Args } from '../src/lib';
+
+/**
+ * Presign a URL for a given key.
+ * @param key
+ * @returns {Promise<string>}
+ */
+async function getPresignedUrlParams( key: string ) : Promise<Args> {
+	const client = new S3Client(
+		{
+			region: process.env.S3_REGION,
+			bucket: process.env.S3_BUCKET,
+		} as S3ClientConfig
```

Comment:
Why is this required? Is there a type error here?

Reply:
Yeah this was copied from the lib.ts. It seems the ts types don't account for this style of passing these times for some reason. Mostly just copied it though